src/main/java/org/obsidiantoaster/generator/rest/HealthResource.java

```diff
@@ -37,4 +54,29 @@ public JsonObject ready()
    {
       return Json.createObjectBuilder().add(STATUS, OK).build();
    }
+   
+   @GET
+   @Path(HealthResource.PATH_CATAPULT_READY)
+   @Produces(MediaType.APPLICATION_JSON)
+   public JsonObject catapultReady() {
+       Client client = ClientBuilder.newBuilder().build();
+       try {
+          WebTarget target = client.target(createCatapultUri());
+          String json = target.request().get().readEntity(String.class);
+          JsonObject object = Json.createReader(new StringReader(json)).readObject();
```

Comment:
Because I also needed to return the error which is a JsonObject. So either I had to figure out how to return a Response in both cases or a JsonObject in both cases. I chose the latter :)